db/__tests__/seed.test.js

```diff
@@ -37,12 +37,52 @@ test('GET 404: user enters invalid endpoint', () => {
 });
 
 describe('/api', () => {
-test("GET 200: responds with an object of endpoints", () => {
+test('GET 200: responds with an object of endpoints"', () => {
     return request(app)
-    .get("/api")
+    .get('/api')
     .expect(200)
-    .then(( { body } ) => {
+    .then(({ body } ) => {
         expect(body).toEqual(endPoints)
     })
 })
+})
+
+describe('/api/articles/:article_id', () => {
+test('GET 200: responds with the article that matches the given article_id', () => {
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then(({ body }) => {
+        expect(body).toMatchObject(
+            {
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: '2020-07-09T20:11:00.000Z', // here the date looks completely different in the test data vs the db table, is it ok that I use the date from the db table rather than convert it somehow from the data?
```

Comment:
you will have this problem with trying to test against dates. there are some jest matchers specifically for dates if you wanted to look into it, you could also do `created_at: expect.any(Date)`